packages/lesswrong/components/posts/FriendlyPostsItem.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { MouseEvent } from "react";
+import { registerComponent, Components } from "../../lib/vulcan-lib";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { usePostsItem, PostsItemConfig } from "./usePostsItem";
+import { SoftUpArrowIcon } from "../icons/softUpArrowIcon";
+import { HashLink } from "../common/HashLink";
+import { useHistory } from "../../lib/reactRouterWrapper";
+import { SECTION_WIDTH } from "../common/SingleColumnSection";
+import withErrorBoundary from "../common/withErrorBoundary";
+import classNames from "classnames";
+
+export const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    maxWidth: SECTION_WIDTH,
+    display: "flex",
+    alignItems: "center",
+    background: theme.palette.grey[0],
+    border: `1px solid ${theme.palette.grey[100]}`,
+    borderRadius: 6, // TODO Use theme.borderRadius.default once it's merged
```

Comment:
Is this now merged?

Reply:
It wasn't when you wrote this, but it is now - fixed!